src/StreamJsonRpc/StreamJsonRpc.csproj

```diff
@@ -3,8 +3,10 @@
     <TargetFrameworks>netstandard2.0;netcoreapp2.1</TargetFrameworks>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
We do *not* want this here, as that significantly slows down the dev's inner loop, which has no reason to block on compiler warnings. We set this in the pipeline itself so that warnings cannot be introduced to the shared repo, but are allowed on local dev boxes.

Reply:
Ok, I will remove it.